love/src/components/WeatherStation/WeatherStation.module.css

```diff
@@ -11,7 +11,7 @@
 
 .section {
   display: grid;
-  grid-template-rows: min-content 1fr;
+  grid-template-rows: min-content minmax(auto, 200px);
```

Comment:
Just wondering about the need of this change?

Reply:
Honestly, I don't know jejeje